cpp/ps0003/150daysprogramming.cpp

```diff
@@ -1,105 +1,21 @@
-// 150daysprogramming.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
-
 #include <iostream>
-using namespace std;
-#include<string>
-
-//bool isFloat(string num);
-
-bool isFloat(string num) {
-    bool isFloating = false;
-    for (char c : num) {
-        if (c == '.') {
-            isFloating = true;
-            break;
-        }
-    }
-    return isFloating;
-}
 
+using namespace std;
 
 int main()
 {
- 
+    // prompting user for number 2/2
     int number;
     cout << " enter the number: ";
     cin >> number;
+

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number